File: Medical_Profile/patients.vb
Imports System

Public Class Patients_Return
 Public Property Token As String
 Public Property [Next] As String
 Public Property Patients As Patient_Return()
 Public Property Totalcount As Integer
End Class

Public Class Patient_Return
 Public Property Preferredname As String
 Public Property Email As String
 Public Property Suffix As String
 Public Property Departmentid As Integer?
 Public Property Primaryproviderid As Integer?
 Public Property Primarydepartmentid As Integer?
 Public Property Homephone As String
 Public Property Insurances As Insurance_Return()
 Public Property Zip As String
 Public Property Lastname As String
 Public Property Address1 As String
 Public Property City As String
 Public Property Lastappointment As DateTime
 Public Property Middlename As String
 Public Property Contactname As String
 Public Property Mobilephone As String
 Public Property Firstname As String
 Public Property State As String
 Public Property Patientid As Integer?
 Public Property Dob As String
 Public Property Contacthomephone As String
 Public Property Contactmobilephone As String
 Public Property Contactrelationship As String
 Public Property Insurance_string As String
End Class

Public Class Insurances_Return
 Public Property Token As String
 Public Property Insurances As Insurance_Return()
 Public Property Totalcount As Integer
End Class

Public Class Insurance_Return
 Public Property Sequencenumber As Integer
 Public Property Insuranceplanname As String
 Public Property Insurancephone As String
End Class

File: Medical_Profile/patients.cs
using global::System;

namespace Medical_Profile
{
 public class Patients_Return
 {
  public string Token { get; set; }
  public string Next { get; set; }
  public Patient_Return[] Patients { get; set; }
  public int Totalcount { get; set; }
 }

 public class Patient_Return
 {
  public string Preferredname { get; set; }
  public string Email { get; set; }
  public string Suffix { get; set; }
  public int? Departmentid { get; set; }
  public int? Primaryproviderid { get; set; }
  public int? Primarydepartmentid { get; set; }
  public string Homephone { get; set; }
  public Insurance_Return[] Insurances { get; set; }
  public string Zip { get; set; }
  public string Lastname { get; set; }
  public string Address1 { get; set; }
  public string City { get; set; }
  public DateTime Lastappointment { get; set; }
  public string Middlename { get; set; }
  public string Contactname { get; set; }
  public string Mobilephone { get; set; }
  public string Firstname { get; set; }
  public string State { get; set; }
  public int? Patientid { get; set; }
  public string Dob { get; set; }
  public string Contacthomephone { get; set; }
  public string Contactmobilephone { get; set; }
  public string Contactrelationship { get; set; }
  public string Insurance_string { get; set; }
 }

 public class Insurances_Return
 {
  public string Token { get; set; }
  public Insurance_Return[] Insurances { get; set; }
  public int Totalcount { get; set; }
 }

 public class Insurance_Return
 {
  public int Sequencenumber { get; set; }
  public string Insuranceplanname { get; set; }
  public string Insurancephone { get; set; }
 }
}